doc/development.rst

```diff
@@ -317,6 +317,8 @@ A root state is a state that has no dependencies.
         def state_B(self):
             pass
 
+The first argument of `@GraphStrategy.depends()` is the default path. If a
+different path should be followed the keyword argument `via` can specify it.
```

Comment:
This should mention that the via argument is to .transition() and not to the .depends() decorator.

Reply:
True.